resources/public/api/conf/1.0/common/errors.yaml

```
@@ -370,4 +370,12 @@ components:
       description: The supplied income source could not be found.
       value:
         code: MATCHING_RESOURCE_NOT_FOUND
-        message: Matching resource not found
\ No newline at end of file
+        message: Matching resource not found
+
+    #422
+
+    requestCannotBeFulfilled:
+      description: Custom (will vary in production depending on the actual error)
```

As per the [tech spec](https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=686162824#:~:text=The%20request%20cannot%20be%20fulfilled.%20More%20specific%20information%20is%20provided%20in%20the%20error%20message.) the description of this error should be as below?

```suggestion
      description: The request cannot be fulfilled. More specific information is provided in the error message.
```